R/feature_methods.R

```diff
@@ -357,8 +357,9 @@ fractionLength <- function(grl, tx_len){
 #' with usually either leaders, cds', 3' utrs or ORFs.
 #' @param RFP RiboSeq reads as GAlignment, GRanges
 #' or GRangesList object
-#' @param GtfOrTx if is a TxDb object of a gtf file, transcripts, called from:
-#'  `exonsBy(Gtf, by = "tx", use.names = TRUE)`, if is a GRL is used as is
+#' @param GtfOrTx if is a TxDb object of a gtf file or transcripts GRangesList
```

Comment:
this is confusing:
If it is link{TxDB  object or a gtf file transcripts will be extracted using \code{exonsBy(Gtf, by = "tx", use.names = TRUE)}, if it is a \link GRangesList already is used as is